src/test/java/org/jpeek/MainTest.java

```diff
@@ -67,6 +67,27 @@ public void crashesIfNoOverwriteAndTargetExists() throws IOException {
         );
     }
 
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
use JUnit5-compatible, OO assertions 
 `new Assertion<>(
            "my text here",
            () -> {
                Main.main(
                    "--sources", Paths.get(".").toString(),
                    "--target", target.toString()
                );
                return null;
            },
            new Throws<>("must throw an exception", IllegalStateException.class)
        ).affirm();`

Reply:
@ilyakharlamov ty, will fix it and also reported #323 to prevent this in future PRs

Reply:
I see you wrapped `buildConsoleAppender` and `buildParameters`, which is probably a good thing.
However there is still this raw `if (!this.quiet) {
            Logger.getRootLogger().removeAppender(console);
        }` which doesn not look right. Something needs to be changed here.

Reply:
![image](https://user-images.githubusercontent.com/5635032/56605743-daa36b00-6615-11e9-825c-14ba1369aa33.png)
![image](https://user-images.githubusercontent.com/5635032/56605761-e5f69680-6615-11e9-9dc4-26746ef9dd50.png)
As you can see - `addAppender` was in the start of a `run` method, now it's the same but inside `buildConsoleAppender` method - while `removeAppender` in the end of the `run` method stays where it was. It's out of scope of my PR to work with it or think about it.

Reply:
I don't see much of improvement in run() method here -  yes, the method became smaller but if (this.quiet) check is now scattered across different methods. You you can rollback `buildConsoleAppender` back if you don't feel like refactoring it.

Reply:
I can't because it's a statically analyze violation of a big complexity in one method, that's why I made these two methods. I think it was a mistake in the initial design. And the only thing I can do in this issue is a new private method and rename `buildConsoleAppender` into `construct` and the new one would be `destruct`

@paulodamaso  - what do you think?